helm/minio/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Multi-Cloud Object Storage
 name: minio
-version: 3.6.3
+version: 3.6.4
```

Comment:
please do not change versions @yisiqi - can you revert?

Reply:
ok, I reverted it.